style/modules/grid.css

```diff
@@ -0,0 +1,20 @@
+.gridContainer{ /* a container with grid */
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-gap: 10px;
+    background: #F5F5F5;
+    padding: 10px;
+}
+
+.gridContainer * {  /* applies to all the items in the grid */
+    background: #508EFF;
+    border-radius: 15px;
+    margin: 10px;
+}
```

Comment:
let's add a blank line after every CSS block (so here, a line between the brace on 13 and the comment on 14)